src/components/NcActions/NcActions.vue

```diff
@@ -1901,9 +1911,9 @@ export default {
 					},
 					on: {
 						show: this.openMenu,
-						'apply-show': this.onOpen,
+						'after-show': this.onOpened,
```

Comment:
This will now run quite late.
`after-show` will already await the next tick and the focus trap, so the additional tick in `onOpened` will cause a 1 tick delay which seems to not be needed

Reply:
I think it's best to stay consistent. I'd like to avoid to have three different timings to watch for. 
Two are perfectly fine:
1. we start closing
2. we are sure everything is finished and closed/hidden